RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -75,7 +75,7 @@
 
 from Configuration.Eras.Modifier_trackingPhase1QuadProp_cff import trackingPhase1QuadProp
 trackingPhase1QuadProp.toModify(detachedQuadStepHitDoublets, layerPairs = [0])
-trackingPhase2PU140.toModify(detachedQuadStepHitDoublets, layerPairs = [0])
+#trackingPhase2PU140.toModify(detachedQuadStepHitDoublets, layerPairs = [0])
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

the same applies to the rest